### PR TITLE
Add CI Checks

### PR DESCRIPTION
**Pull request by joecummings**
- Add Makefile and linting checks
- Satisfy black, flake8, and pylint
- Satisfy PyDocStyle

### CODE REVIEW

File: cdse_covid/pegasus_pipeline/run_amr_parsing_all.py [hunk 1]
```diff
@@ -1,80 +1,79 @@
...
     tokens = spacy_tokenizer(text.strip())
     tokenized_sentence = [token.text for token in tokens]
     return refine_sentence(tokenized_sentence, max_tokens)
 
 
 def refine_sentence(tokenized_sentence: List[str], max_tokens: int) -> List[str]:
-    """
-    If a sentence exceeds the token limit, split the sentence into clauses
-    based on punctuation and keep all tokens within a clause that passes
+    """If a sentence exceeds the token limit, split the sentence into clauses \
```
Comment (elizlee) [inline cdse_covid/pegasus_pipeline/run_amr_parsing_all.py:46]: This seems like a weird docstring format. Is the linter really asking for this?

**Review by elizlee — CHANGES_REQUESTED** [review]
One question + a few other things:
* It looks like the CI tools need to be added to the `requirements`
* Adding `make` instructions to the README might be useful

File: cdse_covid/pegasus_pipeline/run_amr_parsing_all.py (hunk 1 above)
Comment (joecummings) [reply cdse_covid/pegasus_pipeline/run_amr_parsing_all.py:46]: I have thoughts on this, as well. PyDocStyle always wants some sort of punctuation to end the first line (idk why), so you either have to force the sentence to take a shorter form or indicate that the line is still continuing. 

I agree, though, it looks weird.

**Review by elizlee — APPROVED** [review]
Thanks for adding this~
If you want to do something about the PyDocStyle format thing, I don't see harm in adding the configuration `--ignore=D400`

**Pull request merged**